<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="643" started-at="2020-09-07T18:11:32Z" finished-at="2020-09-07T18:11:33Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="643" started-at="2020-09-07T18:11:32Z" finished-at="2020-09-07T18:11:33Z">
      <class name="selenium.DataDriven.ExcelExample">
        <test-method status="SKIP" signature="demoClass(java.lang.String, java.lang.String)[pri:0, instance:selenium.DataDriven.ExcelExample@27efef64]" name="demoClass" duration-ms="0" started-at="2020-09-07T23:41:33Z" data-provider="testdata" finished-at="2020-09-07T23:41:33Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:162)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
	at selenium.DataDriven.ReadExcelFile.getData(ReadExcelFile.java:27)
	at selenium.DataDriven.ExcelExample.testDataExample(ExcelExample.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	... 20 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demoClass -->
      </class> <!-- selenium.DataDriven.ExcelExample -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
